lib/flame/controller.rb

```diff
@@ -156,11 +156,27 @@ def default_path
 			## @example Inherit controller with parent actions by method
 			##   class MyController < BaseController.with_actions
 			##   end
-			def with_actions
+			## @example Include module actions to controller
+			##   class MyController < BaseController
+			##     include with_actions Module1
+			##     include with_actions Module2
+			##     ....
+			##   end
+			def with_actions(mod = nil)
+				return mod.extend(ModuleMixIn) if mod
 				@with_actions ||= Class.new(self) { extend ParentActions }
 			end
 		end
 
+		## Extension to include modules for re-defining methods controller methods
+		module ModuleMixIn
```

Comment:
Extended module is module with actions.

Even before it's extended.

Okay, another one: `IncludingActions`.